express/blocks/columns/columns.css

```diff
@@ -56,8 +56,23 @@ main .columns.center > div {
   align-items: center;
 }
 
-main .columns .brand.icon {
+main .columns .icon-adobe-spark {
```

Comment:
Why did you introduce a new class here? I don't think `.brand.icon` is used for anything else in the mobile viewport.